verticapy/tests_new/performance/vertica/collection/test_import_simple.py

```diff
@@ -55,14 +66,42 @@ def test_no_such_schema(self):
             target_schema="no_such_schema",
             key="no_such_key",
             filename="no_such_file.tar",
-            skip_create_table=True,
         )
-
+        pi.skip_create_table = True
         with pytest.raises(
             ProfileImportError, match=f"Schema no_such_schema does not exist"
         ):
             pi.check_schema()
 
+    def test_missing_bundle(self):
+        """
+        Confirm failure when user specifies a file that does not exist
+        """
+        fname = "no_such_file.tar"
+        pi = ProfileImport(
+            target_schema="schema_not_used", key="no_such_key", filename=fname
+        )
+        pi.skip_create_table = True
+        with pytest.raises(FileNotFoundError, match=f"File {fname} does not exist"):
+            pi.check_file()
+
+    def test_untar_file(self, tmp_path_with_test_bundles):
+        """
+        Confirm failure when user specifies a file that does not exist
```

Comment:
i think the docstring is not accurate. it is not confirming failure.

Reply:
Oops, copy/paste error, I will fix this

Reply:
Fixed this